build_projects/dotnet-cli-build/TestTargets.cs

```diff
@@ -83,7 +83,7 @@ public static BuildTargetResult RestoreTestAssetPackages(BuildTargetContext c)
             CleanNuGetTempCache();
 
             var dotnet = DotNetCli.Stage2;
-            dotnet.Restore("--verbosity", "verbose")
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel")
```

Comment:
Is this a bug in nuget? This will slow down our builds quite a lot, there are tons of these TestAsset projects...